### ISSUE TITLE
New plugin: SAV

### ISSUE THREAD

**Issue by matthdsm** [ewels/MultiQC#1431]
**Plugin Details**

- Name of tool: Illumina SAV
- Tool homepage: https://github.com/CenterForMedicalGeneticsGhent/MultiQC_SAV
- Tool description: replicates some plots and tables from illumina SAV
- Data suitable for MultiQC plot(s): see repo
- Most interesting data for General Stats table: None

**Tool output:**
MultiQC sections

** Remarks **
This plugin can be forked into the MultiQC organisation. I think it's good to go, just needs a review.

**Comment by matthdsm** [comment]
Hi Phil,

Friendly bi-weekly (or something) ping to bug you into merging this.

Cheers
Matthias

**Comment by ewels** [comment]
Haven't had a chance to review but have forked anyway: https://github.com/MultiQC/MultiQC_SAV (better to have it there).

Will leave this issue open as a reminder to review 👍🏻

**Comment by matthdsm** [comment]
Great! Can you add me as collaborator? Then I can update the readme en setup.

M

**Comment by ewels** [comment]
Of course, added 👍🏻

**Comment by matthdsm** [comment]
Thanks!

**Comment by matthdsm** [comment]
Hi Phil,

Friendly bi-weekly ping to keep this on the radar 😉 
If you need me to stop bugging you, feel free to let me know.

Cheers
M

**Comment by ewels** [comment]
Right! Yes ok so realistically I don't think I'm going to get to this for a while yet sorry - about to head off for vacation and then will be off with a new baby. Should be back around November time so will try to take a look then. So feel free to hit pause for a little while but then start it up again towards the end of the year 😄 

Let me know if you need anything specific with this in a shorter time frame and I'll see what I can do 👍🏻

**Comment by matthdsm** [comment]
Hey, congrats!
Sounds like you're going to need the time off 😉 

I'll hold off bugging you until the end of the year then. In the mean time, I can always fix some stuff myself if needed.

Have fun on vacation and with the new baby!

M

**Comment by ewels** [comment]
Thanks! 😁

**Comment by matthdsm** [comment]
Hi Phil,

Hope all is well with you and the baby. If you find time, would you be able to review the plugin?

Thanks
Matthias
